capsicum-test-main.cc

```diff
@@ -27,6 +30,40 @@ class SetupEnvironment : public ::testing::Environment
     }
     std::cerr << tmpdir << std::endl;
   }
+  void CheckCapsicumSupport() {
+    // For versions of googletest that lack GTEST_SKIP.
+#ifndef GTEST_SKIP
+#define GTEST_SKIP GTEST_FAIL
+#define GTEST_SKIP_defined
+#endif
+
+#ifdef __FreeBSD__
+    size_t trap_enotcap_enabled_len;
+    int rc;
+    bool trap_enotcap_enabled;
+
+    trap_enotcap_enabled_len = sizeof(trap_enotcap_enabled);
```

Comment:
nit: this chunk could move down a few lines closer to first use, and maybe be compacted a bit:
```c++
bool trap_enotcap_enabled;
size_t trap_enotcap_enabled_len = sizeof(trap_enotcap_enabled);
int rc = sysctlbyname("kern.trap_enotcap", &trap_enotcap_enabled, &trap_enotcap_enabled_len, nullptr, 0);
```

Reply:
Done.

Reply:
super-nit: could still move the declarations down next to the declaration of `oid` maybe?